book/website/reproducible-research/rdm/rdm-dmp.md

```diff
@@ -0,0 +1,74 @@
+Defining who is responsible for the management of the data and code can prevent confusion/miscommunication later in the project.
+    * You should check the requirements of your institute and funder. 
+The library research support team of your institute and the website of your funder are usually good places to find information and help. 
+Some of the funders require you to use their Data Management Plan template. 
+You can check if your funder or institute has a Data Management Plan template using [DMPonline](https://dmponline.dcc.ac.uk/).
+2. **Type and size of data collected and documentation/metadata generated**
+    * You can list the file formats you will be using to collect, process and present your data. 
+If you want to share your outputs later on, open file formats that can be opened and used without a particular license for a software programme are preferred, so you may have to convert your files or start working in these formats early on. 
+    * A distinction can be made between different types of data that can be described in the plan separately: 
+        * Raw/primary data: data collected from the source (always keep a read-only version of raw data so you can come back to it later!)
+        * Processed data: a version of the data that has been modified for analysis or visualisation
+        * Finalised data: data that is ready to be shared in a publication or data repository (see {ref}`Sharing and archiving data section <rr-rdm-sharing>` for more information)
+    * All of these types of data will have to be described to be placed into context by using metadata (see the {ref}`Documentation and metadata section<rr-rdm-metadata>`) and adequate documentation which will allow future you, and anyone in your team, to interpret the data. 
+    * It is helpful to know the approximate size (in the range of MB, GB, TB or PB) of the data in these various stages, because this will affect the storage solutions available for you (see below). 
+3. **Type of data storage used and back up procedures that are in place** 
+    * Check the {ref}`Data storage and organisation section<rr-rdm-storage>` for storage and back-up solutions and ways to organise your files 
+    * Check if there are any **costs** associated with your project
+        * Preferred storage solution (during and after the project, see below)
+        * Personnel costs (if you need a data manager to manage more sensitive or large quantities of data)
+        * Software licenses (such as Electronic Lab Notebooks, see the {ref}`Open notebooks section<rr-open-notebooks>`
+        * You can use this [checklist for costs](https://www.ukdataservice.ac.uk/media/622368/costingtool.pdf) as a guidance
+    * Keeping track of who changed what in your data/code can be important, particularly for code.
+See the {ref}`Version Control chapter<rr-vcs>` for more information.
+    * Determine who has access to the data. 
```

Comment:
It will be useful to talk about who grants access to the data?

Reply:
I think that's going to be dependent on the project. We discuss data access more in depth in the personal data section but I don't think I can refer to that section yet!